buf/registry/module/v1/download_service.proto

```diff
@@ -48,7 +48,10 @@ message DownloadRequest {
     // Specific file types to request.
     //
     // If not set, all file types are returned.
-    repeated FileType file_types = 2 [(buf.validate.field).repeated.items.enum.defined_only = true];
+    repeated FileType file_types = 2 [
+      (buf.validate.field).repeated.items.enum.defined_only = true,
+      (buf.validate.field).repeated.items.enum.not_in = 0
```

Comment:
Should we add the unique constraint here too?

Reply:
Agreed, added here and to `v1beta1`